build.sbt

```diff
@@ -555,6 +559,62 @@ lazy val coordination = akkaModule("akka-coordination")
   .settings(AutomaticModuleName.settings("akka.coordination"))
   .settings(OSGi.coordination)
 
+lazy val `maven-dependencies` = Project(id="maven-dependencies",base= file("maven-dependencies"))
+  .enablePlugins(HeaderPlugin)
+  // buildSettings and defaultSettings configure organization name, licenses, etc...
+  .settings(AkkaBuild.buildSettings)
+  .settings(AkkaBuild.defaultSettings)
+  .settings(
+    name := "akka-maven-dependencies",
```

Comment:
How about `akka-bom` named?

Reply:
Yeah! I started off following Lagom's naming but we already discussed this naming improvement in the Akka-HTTP (https://github.com/akka/akka-http/pull/3665) PR.

Goo catch.

Reply:
As I learned later (see https://github.com/playframework/playframework/pull/10549#issuecomment-733808616), `foo-bom` and `foo-dependencies` are convention names for two different types of BOMs.